cEP-0019.md

```diff
@@ -0,0 +1,404 @@
+[GSoC'18 project](https://summerofcode.withgoogle.com/projects/#5188493739819008).
+
+## Background
+
+People respond to review comments on pull requests by giving feedback. Emojis
+make it much easier to give and receive feedback without a long comment thread.
+Those emojis are called
+[reactions](https://developer.github.com/v4/enum/reactioncontent/) on GitHub
+or [award emoji](https://docs.gitlab.com/ee/user/award_emojis.html) on GitLab.
+THUMBS_UP and THUMBS_DOWN, together with other reactions, are used by people
+especially the author of pull request to publicly acknowledge or reject review
+comments. This provides feedback to reviewers and all other members.
+
+GitLab is beyond the scope of this project, as GitHub and GitLab have quite
+different APIs and most core projects of coala are based on GitHub. This
+proposal will only discuss reactions on GitHub, but not award emoji on GitLab.
+
+There are three types of reviews on a pull request:
+
+1. Issue Comment
+   [see sample](https://github.com/coala/gh-board/pull/18#issuecomment-385878735)
+
+2. Review Summary
+   [see sample](https://github.com/coala/gh-board/pull/18#pullrequestreview-116783753)
+
+3. Review Discussion
+   [see sample](https://github.com/coala/gh-board/pull/18#discussion_r186249423)
+
+However, those reactions spread over all PRs and are not collected and
+analyzed. Moreover, GitHub doesn't send notifications to reviewers who receive
+reactions. By tracking reactions, a nice feedback loop would be created
+for both senior and junior reviewers.
+
+## Introduction
+
+The meta-review project is about making use of reactions to build a
+meta-review system, similar to a meta-moderation system. These responses
+are to be collected, processed and displayed on
+[gh-board](https://github.com/coala/gh-board), which is a nice kanban
+board. Compared to [Huboard](https://huboard.com/) which requires server-side
+support, gh-board is serverless.
+
+The scope of this project would exclude non-public meta-reviews. That is,
+there would not be any anonymous meta-reviews as that introduces too many
+additional complexities. The public only version would be used for
+a long time before adding support for non-public meta-reviews.
+
+Note there is a technical limitation in that reactions can not be placed
+onto the "review summary", which means general review comments should not
+be put into the "review summary" text box, as it will be not able to be
+meta-reviewed. Instead, put general review comment as a comment like
+[this](https://github.com/coala/gh-board/pull/18#issuecomment-385925740).
+Only put comments on "review summary" if it is indeed a summary of "review
+discussions".
+
+## Objectives
+
+One goal of this project is to encourage people to do meta-reviews,
+systematically. Meta-reviews in need will be displayed on gh-board. The
+implementation details are discussed in the next section.
+
+Also, to encourage people to do more meta-reviews, statistics of meta-reviews
```

Comment:
IMO, we can make an objective : update newcomer process to require potential developers have done at least 'x' reviews, and have done at least one meta-review, and have received a meta-review score of 'y'.

That type of objective, once accepted as part of a cEP, ensures the org uses your work, and thus they have an interest in ensuring your work helps the org ('buy-in').

Reply:
Done below

Reply:
Could you elaborate on that? This should be done somewhere below?

Reply:
no, im just marking this comment as already done, since this review comment hasnt been automatically dismissed.